src/core/electric_utility.jl

```diff
@@ -195,6 +200,51 @@ struct ElectricUtility
         cambium_emissions_region = "NA - Cambium data not used for climate emissions" # will be overwritten if Cambium is used
         
         if !is_MPC
+            # Initialize clean energy fraction series
+            clean_energy_series_dict = Dict{String, Union{Nothing, Array{<:Real, 1}}}()
+            if typeof(clean_energy_fraction_series) <: Real  # user provided scalar value
+                clean_energy_series_dict["cef"] = repeat([clean_energy_fraction_series], 8760*time_steps_per_hour)
+            elseif length(clean_energy_fraction_series) == 1  # user provided array of one value
+                clean_energy_series_dict["cef"] = repeat(clean_energy_fraction_series, 8760*time_steps_per_hour)
+            elseif length(clean_energy_fraction_series) / time_steps_per_hour ≈ 8760  # user provided array with correct length
+                clean_energy_series_dict["cef"] = clean_energy_fraction_series
+            elseif length(clean_energy_fraction_series) > 1 && !(length(clean_energy_fraction_series) / time_steps_per_hour ≈ 8760)  # user provided array with incorrect length
+                if length(clean_energy_fraction_series) == 8760
+                    clean_energy_series_dict["cef"] = repeat(clean_energy_fraction_series, inner=time_steps_per_hour)
+                    @warn("Clean energy fraction series has been adjusted to align with time_steps_per_hour of $(time_steps_per_hour).")
+                else
+                    throw(@error("The provided ElectricUtility clean energy fraction series does not match the time_steps_per_hour."))
+                end
+            else
+                # Retrieve clean energy fraction data if not user-provided
+                if cambium_start_year < 2023 || cambium_start_year > 2050
+                    @warn("The cambium_start_year must be between 2023 and 2050. Setting cambium_start_year to 2024.")
+                    cambium_start_year = 2024 # Must update annually 
+                end
+                try
+                    clean_energy_response_dict = cambium_clean_energy_fraction_profile(
+                        scenario = cambium_scenario, 
+                        location_type = cambium_location_type, 
+                        latitude = latitude, 
+                        longitude = longitude,
+                        start_year = cambium_start_year,
+                        lifetime = cambium_levelization_years,
+                        metric_col = cambium_cef_col,
+                        grid_level = cambium_grid_level,
```

Comment:
@pypapus could you check if changing the grid_level changes the values returned for the cef_load metric?

Reply:
Changing the grid level from "enduse" to "busbar" does not change the value of the cef_load.